src/adr/0032-distributed-mail-queue-cleanup.md

```diff
@@ -30,8 +30,18 @@ Failing cleanup will lead to the content being eventually updated upon next `bro
 
 ## Consequences
 
-MailQueue content will eventually be dropped both in Cassandra and in ObjectStorage once Deduplicated Blob Store garbage 
+MailQueue content will eventually be dropped both in Cassandra and in ObjectStorage once Blob Store garbage 
```

Comment:
You misinterpreted my comment: the BlobStore has a delete method. It happens that we have an deduplicating implementation of it but this ADR should not care at all about whether this implementation exists, at all.
From MailQueue point of view, we should just care about deleting blobs we don't use anymore.

Reply:
That's why I remove "deduplicated"